apps/example-tailwind/src/app/pages/page-profile/page-profile.component.html

```diff
@@ -1,25 +1,712 @@
+                                            id="taxId"
+                                            type="taxId"
+                                            pInputText
+                                            name="taxId"
+                                            [(ngModel)]="user.taxId"
+                                            [disabled]="isAddressDisabled"
+                                            [placeholder]="user.taxId" />
+                                    </div>
+                                </div>
+
+                                <div class="p-field mt-2 flex justify-end">
+                                    <button
+                                        type="submit"
+                                        pButton
+                                        label="Save"
+                                        class="bg-green ml-5 rounded-sm px-4 py-1 text-sm text-white"
+                                        *ngIf="!isAddressDisabled"></button>
+                                </div>
+                            </form>
+                        </div>
+                    </p-card>
+                </div>
+            </div>
+        </div>
+        <div class="w-full md:w-2/5">
+            <div>
+                <div>
+                    <p-card>
+                        <h2 class="mb-3 text-lg font-bold">Change Password</h2>
+                        <form
+                            [formGroup]="changePasswordFormService.form"
+                            (ngSubmit)="onSubmit()">
+                            <div class="mb-2">
+                                <label
+                                    for="current password"
+                                    class="mb-2 block text-sm font-semibold"
+                                    >Current Password</label
+                                >
+                                <input
+                                    class="h-8 w-full rounded-md border-[1.5px] p-2 placeholder:text-sm"
+                                    placeholder="Current Password"
+                                    type="password"
+                                    formControlName="currentPassword" />
+                                <div
+                                    *ngIf="
+                                        changePasswordFormService.form.get(
+                                            'currentPassword'
+                                        )!.touched &&
+                                        changePasswordFormService.form.get(
+                                            'currentPassword'
+                                        )!.dirty &&
+                                        changePasswordFormService.form.get(
+                                            'currentPassword'
+                                        )!.errors as errors
+                                    "
```

Comment:
convert it to modern syntax 
```suggestion

@if(changePasswordFormService.form.get(
                                            'currentPassword'
                                        )!.touched &&
                                        changePasswordFormService.form.get(
                                            'currentPassword'
                                        )!.dirty &&
                                        changePasswordFormService.form.get(
                                            'currentPassword'
                                        )!.errors as errors)
                                <div>
                                  
                                        
                                    "
```